ecosystem/typescript/sdk/src/tests/e2e/indexer.test.ts

```diff
@@ -9,19 +8,17 @@ import { Network, NetworkToIndexerAPI, sleep } from "../../utils";
 import { Provider } from "../../providers";
 import { AptosToken } from "../../plugins";
 
-const provider = new Provider(Network.TESTNET);
+const provider = new Provider(Network.DEVNET);
 const aptosToken = new AptosToken(provider);
-const faucetClient = new FaucetClient(
-  "https://fullnode.testnet.aptoslabs.com",
-  "https://faucet.testnet.aptoslabs.com",
-  { TOKEN: FAUCET_AUTH_TOKEN },
-);
+const faucetClient = new FaucetClient("https://fullnode.devnet.aptoslabs.com", "https://faucet.devnet.aptoslabs.com", {
+  TOKEN: FAUCET_AUTH_TOKEN,
+});
```

Comment:
I think we should do all Indexer tests on devnet (while we dont have localnet) so it can wiped out. now that this test is not land-blocking in CI we can use devnet

Reply:
Okay if we do that can you add a `before` step to every test that checks that devnet is actually up and give a super clear explanation if it is not? If devnet is being released these tests will fail which will be confusing for people.